force-app/main/default/classes/tasks/CollectionsTasks.cls

```diff
@@ -6,7 +6,17 @@ public class CollectionsTasks {
   public static Integer countZerosInBitList(
     List<Integer> listWithZerosAndUnits
   ) {
-    return 0;
+    
+    listWithZerosAndUnits.sort();
+    System.debug(listWithZerosAndUnits);
+    Integer indexOfFirstUnit = listWithZerosAndUnits.indexOf(1);
```

Comment:
What if you try to sum all of the elements in such array - does it give you additional information?

Reply:
Updated. Thanks, it totally works.